### ISSUE TITLE
[feature request] allow granular control over prerelease flag in github

### ISSUE THREAD

**Issue by ahmadnassri** [release-it/release-it#679]
currently the `prerelease` flag on github releases is driven by [parsing the version number](https://github.com/release-it/release-it/blob/master/lib/plugin/github/GitHub.js#L182), however, this does not accurately mirror the functionality by github itself. 

the flag on github is simply a flag, and has no relation to the version number:

![image](https://user-images.githubusercontent.com/183195/85359562-9abf6880-b4e4-11ea-81b9-a61e5cd4af3d.png)

the use-case here would be to allow incrementing semver numbers normally, without relying on suffix values or requiring a follow up increment to move from "prerelease" to "produciton" which in github is an action of unchecking the box

**Comment by webpro** [comment]
Yes, there is the `--github.preRelease` setting for this.

(Setting e.g. `--preRelease=next` sets this automatically indeed.)

**Comment by ahmadnassri** [comment]
in my testing doing `--github.preRelease` does nothing by itself (and looking at the code, it seems to also confirm that, unless i'm misreading it)

using `--preRelease` does indeed work, but then it overwrites the version number to do a "prerelease" as per semver standard ...

so what i'm looking for is being able to check the box in github, without modifying the version numbering scheme

**Comment by webpro** [comment]
Right, in that case it's a bug, unfortunately. `--github.preRelease` should just work.
